### ISSUE TITLE
Bad, dangerous, and unused redefinition of skos: prefix

### ISSUE THREAD

**Issue by rivettp** [iofoundry/ontology#95]
In Metadatadefinition and Metadatasupplychain.
For example:
 @prefix skos: <http://www.w3.org/2004/02/skos/supplychain#> .

It might seem trivial if not used but if importing this file into a triple store with the option to onboard prefixes from the import file this could really screw up the whole system!

**Comment by ekwallace** [comment]
I agree with Pete that there is a bug in the IRI assigned for the skos entity and prefix in lines 9 and 20 respectively of Metadatasupplychain.rdf.  They should both use  "http://www.w3.org/2004/02/skos/core#"
I wouldn't be surprised if there were more typos like this in the rdf files supporting the infrastructure but I couldn't find a Metadatadefinition file.  
Pete: Where is Metadatadefinition?
